lib/init.coffee

```diff
@@ -113,22 +147,31 @@ module.exports =
         ///g
       reResult = re.exec(toParse)
       while reResult?
-        ret.push
-          type: "Warning"
-          text: reResult[3]
-          filePath: path.join(projectPath(textEditor), reResult[1])
-          range: helpers.rangeFromLineNumber(textEditor, reResult[2] - 1)
+        try
+          ret.push
+            type: "Warning"
+            text: reResult[3]
+            filePath: path.join(projectPath(textEditor), reResult[1])
+            range: new Range(
+              [reResult[3] - 1, 0],
+              [reResult[3] - 1, 0]
+            )
+        catch Error
+          console.log("Elixir Linter Error", Error)
         reResult = re.exec(toParse)
       ret
 
+
     handleResult = (textEditor) ->
       (compileResult) ->
         resultString = compileResult['stdout'] + "\n" + compileResult['stderr']
         try
           errorStack = parseError(resultString, textEditor)
           warningStack = parseWarning(resultString, textEditor)
+          legacyWarningStack = parseLegacyWarning(resultString, textEditor)
           (error for error in errorStack.concat(warningStack) when error?)
         catch Error
+          console.log(Error, "oops caught error....")
```

Comment:
Prefix this with an identifier as to where it is coming from (`linter-elixirc: `).